godot/export_presets.cfg

```diff
@@ -2,13 +2,13 @@
 
 name="linux"
 platform="Linux/X11"
-runnable=true
+runnable=false
```

Comment:
not sure why is this change

Reply:
This is the "toggle" of which preset are you using for the same platform.

For example, if you have multiple Linux exports, you choose one.
Also with Android, if you have multiple Android exports (one for Android Mobile and another for Android VR, you need to toggle which is the default).

I fixed it, but I left the Android as the default between Android and Quest.